pom.xml

```diff
@@ -99,11 +99,6 @@ THE SOFTWARE.
     <!-- Make sure to keep the jetty-ee9-maven-plugin version in war/pom.xml in sync with the Jetty release in Winstone: -->
     <winstone.version>8.1</winstone.version>
     <node.version>20.17.0</node.version>
-    <!-- frontend-maven-plugin will install this Yarn version as bootstrap, then hand over control to Yarn Berry. -->
-    <yarn.version>1.22.19</yarn.version>
```

Comment:
Yarn 1.x is now no longer needed to bootstrap Yarn Berry, since we now use Corepack for this purpose.